Web_Development/FrontEnd/CSS_Comment/CSS_Comments.md

```diff
@@ -0,0 +1,57 @@
+
+## CSS Comments
+Comments are used to explain the code, and may help when you edit the source code at a later date.
+A CSS comment is used to add explanatory notes to the code or to prevent the browser from interpreting specific parts of the style sheet. By design, comments have no effect on the layout of a document.
+The /* */ comment syntax is used for both single and multiline comments. There is no other way to specify comments in external style sheets.

```

Comment:
Make it inside inline code: `/* */` like this.

Reply:
done